src/psyclone/tests/psyir/frontend/fparser2_part_ref_test.py

```diff
@@ -99,6 +94,48 @@ def test_handling_part_ref_expression():
     assert len(new_node.children) == 3  # Array dimensions
 
 
+def test_handling_part_ref_without_symbol_type(fortran_reader):
+    '''Test that fparser2 Part_Ref is converted to a Call if the reference
+    symbol if of unknown kind. Unless it is found on a specific location that
```

Comment:
"is" and best to remove the full stop before "Unless"

Reply:
Done both